File: app/src/main/java/proj/kolot/com/nearshops/retrofit/NearbyApi.java
package proj.kolot.com.nearshops.retrofit;


import proj.kolot.com.nearshops.data.GeneralResponse;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface NearbyApi {

    @GET("maps/api/place/nearbysearch/json")
    Call<GeneralResponse> getNearByPlaces(@Query("location") String latLong, @Query("radius") int radius, @Query("type") String type, @Query("key") String key);


}

File: app/src/main/java/proj/kolot/com/nearshops/retrofit/NearbyApi.kt
package proj.kolot.com.nearshops.retrofit

import proj.kolot.com.nearshops.data.GeneralResponse
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface NearbyApi {

    @GET("maps/api/place/nearbysearch/json")
    fun getNearByPlaces(@Query("location") latLong: String, @Query("radius") radius: Int, @Query("type") type: String, @Query("key") key: String): Call<GeneralResponse>


}
